src/rsa.rs

```diff
@@ -151,7 +98,43 @@ rsa_pkcs1_padding!(RSA_PKCS1_SHA512, &digest::SHA512,
                    &SHA512_PKCS1_DIGESTINFO_PREFIX,
                    "Signing using RSA with PKCS#1 1.5 padding and SHA-512.");
 
-macro_rules! rsa_pkcs1 {
+rsa_pss_padding!(RSA_PSS_SHA1, &digest::SHA1,
+                   "Signing using RSA with PSS padding and SHA-1.", 0);
+rsa_pss_padding!(RSA_PSS_SHA256, &digest::SHA256,
+                   "Signing using RSA with PSS padding and SHA-256.", 0);
+rsa_pss_padding!(RSA_PSS_SHA384, &digest::SHA384,
+                   "Signing using RSA with PSS padding and SHA-384.", 0);
+rsa_pss_padding!(RSA_PSS_SHA512, &digest::SHA512,
+                   "Signing using RSA with PSS padding and SHA-512.", 0);
+
+rsa_pss_padding!(RSA_PSS_SHA1_20, &digest::SHA1,
+                   "Signing using RSA with PSS padding, 20 byte salt, and SHA-1.", 20);
+rsa_pss_padding!(RSA_PSS_SHA256_20, &digest::SHA256,
+                   "Signing using RSA with PSS padding, 20 byte salt, and SHA-256.", 20);
```

Comment:
You can drop the ", XXX byte salt, " part. Instead, in the boilerplate for PSS, we can state that the salt length is equal to the digest length and the MGF digest is the same as the digest function used to hash the message.